docs/standard/garbage-collection/large-object-heap.md

```diff
@@ -0,0 +1,313 @@
+
+![Figure 1: A gen 0 GC and a gen 1 GC](media/loh/loh-figure-1.jpg)   
+Figure 1: A generation 0 and a generation 1 GC.
+
+Figure 2 shows that after a generation 2 GC which saw that `Obj1` and `Obj2` are dead, the GC forms contiguous free space out of memory that used to be occupied by `Obj1` and `Obj2`, which then was used to satisfy an allocation request for `Obj4`. The space after the last object, `Obj3`, to end of the segment can also be used to satisfy allocation requests.
+ 
+![Figure 2: After a gen 2 GC](media/loh/loh-figure-2.jpg)  
+Figure 2: After a generation 2 GC
+
+If there isn't enough free space to accommodate the large object allocation requests, the GC first attempts to acquire more segments from the OS. If that fails, it triggers a generation 2 GC in the hope of freeing up some space.
+
+During a generation 1 or generation 2 GC, the garbage collector releases segments that have no live objects on them back to the OS by calling the [VirtualFree function](https://msdn.microsoft.com/library/windows/desktop/aa366892(v=vs.85).aspx). Space after the last live object to the end of the segment is decommitted. And the free spaces remain committed though they are reset, meaning that the OS doesn’t need to write data in them back to disk. 
+
+Since the LOH is only collected during generation 2 GCs, the LOH segment can only be freed during such a GC. Figure 3 illustrates a scenario where the garbage collector releases one segment (segment 2) back to the OS and decommits more space on the remaining segments. If it needs to use the decommitted space at the end of the segment to satisfy large object allocation requests, it commits the memory again. (For an explanation of commit/decommit, see the documentation for [VirtualAlloc](https://msdn.microsoft.com/library/windows/desktop/aa366887(v=vs.85).aspx).
+ 
+![Figure 3: LOH after a gen 2 GC](media/loh/loh-figure-3.jpg)  
+Figure 3: The LOH after a generation 2 GC
+
+## When is a large object collected?
+
+In general, a GC occurs when one of the following following 3 conditions happens:
+
+- Allocation exceeds the generation 0 or large object threshold.
+
+   The threshold is a property of a generation. A threshold for a generation is set when the garbage collector allocates objects into it. When the threshold is exceeded, a GC is triggered on that generation. When you allocate small or large objects, you consume generation 0 and the LOH’s thresholds, respectively. When the garbage collector allocates into generation 1 and 2, it consumes their thresholds. These thresholds are dynamically tuned as the program runs.
+
+   This is the typical case; most GCs happen because of allocations on the managed heap.
+
+- The <xref:System.GC.Collect%2A?displayProperty=nameWithType> method is called.
```

Comment:
this just displays as "The xref:System.GC.Collect%2A?displayProperty=nameWithType method is called." when I look at it as a doc (ie not mark down) - is that expected?